server/engine/src/engine.py

```diff
@@ -56,26 +56,37 @@ def main():
         # The kinematics fit will fit the body scales, marker offsets, and motion of the subject, to all the trial
         # segments that have not yet thrown an error during loading.
         print('Running kinematics fit', flush=True)
-        subject.run_kinematics_fit(DATA_FOLDER_PATH)
+        subject.run_kinematics_pass(DATA_FOLDER_PATH)
         # This will create a B3D object in memory for the current fit of the subject. This can be used at any point to
         # write out the B3D file, but also can be used as our working object as we run subsequent pipeline steps.
         subject_on_disk: nimble.biomechanics.SubjectOnDisk = subject.create_subject_on_disk(href)
 
-        print('Running acc minimizing pass...', flush=True)
-        add_acc_minimize_pass(subject_on_disk)
+        print('Running acceleration minimizing pass...', flush=True)
+        print('-> This pass runs a simple least-squares optimization to minimize the acceleration of each DOF for the '
+              'subject over each trial, subject to also tracking the original trajectory closely. This is akin to a '
+              'Butterworth lowpass filtering step, except that it is better able to ensure that the high frequency '
+              'noise in the finite-differenced accelerations is knocked down, which makes the torque plots smoother.', flush=True)
+        add_acceleration_minimizing_pass(subject_on_disk)
 
         print('Detecting missing GRF frames...', flush=True)
+        print('-> This runs a set of heuristics to detect frames in the input data where we believe the subject is in '
+              'contact with the ground, but there is no force plate data. These heuristics are deliberately a bit too '
+              'aggressive in marking frames as "missing GRF", in order to make sure we get almost all the frames where '
+              'there actually is missing GRF labeled correctly. This means the dataset of frames which are marked as '
+              'not missing GRF are going to be very clean, but smaller than we might have with more selective '
+              'heuristics.', flush=True)
```

Comment:
Would this be a good place to make a note to users that they can manually mark frames as "good"?